appcircle_firebase_distribution/1.0.2/component.yaml

```diff
@@ -0,0 +1,73 @@
+platform: Common
+buildPlatform:
+displayName: Firebase App Distribution
+description: "Distribute your builds via Firebase App Distribution"
+docsUrl: https://docs.appcircle.io/workflows/common-workflow-steps#firebase-app-distribution
+requiredComponents: "appcircle_ios_build_sign,appcircle_android_build"
+precedingComponents: 
```

Comment:
Other build steps that give app output should also be added here.

Reply:
on the ios side only appcircle_ios_sign component generates ipa that user can use. The other generating ipa steps are related to 3rd party test automation tools and user can not use directly to them.